wordpress-mobile/gutenberg-mobile
# Bump Gutenberg reference to `5194cdb`

Bumps the Gutenberg reference to `5194cdb`.

This PR includes three fixes that were required to include recent changes from Gutenberg repo:
1. Disable NPM version 7 - https://github.com/wordpress-mobile/gutenberg-mobile/pull/3307
2. Fix: Size setting is not displayed - https://github.com/WordPress/gutenberg/pull/30186
3. Update the `package-lock.json` file due to [recent changes in `gutenberg/package.json`](https://github.com/WordPress/gutenberg/pull/30018).

To test:
**For testing this PR, verify first that the installed NPM version is version 6.x.**

1. Run `npm install`.
2. Verify that the `package-lock.json` file remains unchanged.

PR submission checklist:
